server/src/main/java/org/elasticsearch/health/HealthIndicatorResult.java

```diff
@@ -20,9 +21,21 @@ public record HealthIndicatorResult(
     HealthStatus status,
     String summary,
     HealthIndicatorDetails details,
-    List<HealthIndicatorImpact> impacts
+    List<HealthIndicatorImpact> impacts,
+    @Nullable List<UserAction> userActions
```

Comment:
I'm on the fence between making this nullable (so it doesn't show up in the output if there are none) vs making it always there but empty (for ease of use in the code)